ComfortZoneApp/app/src/main/java/com/example/comfortzone/Fragments/InputFragment.java

```diff
@@ -33,6 +35,8 @@ public class InputFragment extends Fragment {
     private WeatherData weatherData;
     private Button btnSubmit;
     private Slider slComfortLevel;
+    private ParseUser currentUser;
+    private int temp;
```

Comment:
I would caution using local variables in activities/fragments because you want to make sure you are restoring them when there's config change (i.e. portrait/landscape change, light/dark mode, language change). local variables basically get reset on config change, and you might see unintended behavior